lib/host.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, pool: Pool, hostname_or_ip):
         self.xcp_version_short = f"{self.xcp_version.major}.{self.xcp_version.minor}"
         self._dom0: Optional[VM] = None
 
+        self._check_bug_reports()
```

Comment:
A Host object could be constructed at any time. So this'd be better placed near e.g. `track_test_host()` to avoid issuing duplicate warnings.